recipes/CommonVoice/ASR/transformer/train.py

```diff
@@ -269,15 +269,21 @@ def dataio_prepare(hparams, tokenizer):
         replacements={"data_root": data_folder},
     )
     # We also sort the validation data so it is faster to validate
-    valid_data = valid_data.filtered_sorted(sort_key="duration")
+    valid_data = valid_data.filtered_sorted(
+        sort_key="duration",
+        key_max_value={"duration": hparams["avoid_if_longer_than_val_test"]},
+    )
 
     test_data = sb.dataio.dataset.DynamicItemDataset.from_csv(
         csv_path=hparams["test_csv"],
         replacements={"data_root": data_folder},
     )
 
     # We also sort the validation data so it is faster to validate
-    test_data = test_data.filtered_sorted(sort_key="duration")
+    test_data = test_data.filtered_sorted(
+        sort_key="duration",
+        key_max_value={"duration": hparams["avoid_if_longer_than_val_test"]},
+    )
```

Comment:
I think we should definitely add a warning/log saying that we are skipping files that have a duration > to `avoid_if_longer_than_val_test`.

Reply:
Do it :p